src/pages/markets/basket-indices/index.tsx

```diff
@@ -3,8 +3,9 @@ import BasketIndices from '../components/markets/_basket-indices'
 import { DerivedFXHero } from '../components/sections/_hero_derived_fx'
 import NavTab from '../components/sections/_nav-tab'
 import { simple_step_content_basket } from '../static/content/_basket'
+import Signup, { Appearances } from 'components/custom/signup'
 import Layout from 'components/layout/layout'
-import { localize, WithIntl } from 'components/localization'
+import { localize, Localize, WithIntl } from 'components/localization'
```

Comment:
please remove `Localize`, it's not used at all.

Reply:
thanks @michio-rework , sorry, not correct resolve conflicts